src/IntegralArrays.jl

```diff
@@ -1,5 +1,74 @@
 module IntegralArrays
 
-# package code goes here
+using FileIO # for loading images
+using QuartzImageIO, ImageMagick, ImageSegmentation, ImageFeatures # for reading images
+using Colors # for making images greyscale
+using Images # for channelview; converting images to matrices
+using ImageTransformations # for scaling high-quality images down
+
+#=
+Any one pixel in a given image has the value that is the sum of all of the pixels above it and to the left.
+Original    Integral
++--------   +------------
+| 1 2 3 .   | 1  3  6 .
+| 4 5 6 .   | 5 12 21 .
+| . . . .   | . . . . .
+=#
+
+function getImageMatrix(imageFile::AbstractString)
+    img = load(imageFile)
+    #img = imresize(img, ratio=1/8)
+
+    # imgArr = convert(Array{Float64}, channelview(img)) # for coloured images
+    imgArr = convert(Array{Float64}, Colors.Gray.(img))
+    
+    return imgArr
+end
+
+function toIntegralImage(imgArr::AbstractArray)
+    
+    arrRows, arrCols = size(imgArr) # get size only once in case
+    rowSum = zeros(arrRows, arrCols)
+    integralImageArr = zeros(arrRows, arrCols)
+    
+    # process the first column
+    for x in 1:(arrRows)
```

Comment:
```suggestion
    for x in 1:arrRows
```

However, to support arbitrary axes (https://julialang.org/blog/2017/04/offset-arrays/) it's best to say `for x in axes(imgArr, 1)`